react/features/base/media/middleware.js

```diff
@@ -8,8 +8,8 @@ import {
 } from '../../analytics';
 import { isRoomValid, SET_ROOM, setAudioOnly } from '../conference';
 import JitsiMeetJS from '../lib-jitsi-meet';
-import { getPropertyValue } from '../profile';
 import { MiddlewareRegistry } from '../redux';
+import { getPropertyValue } from '../settings';
```

Comment:
I want to make sure I got this right. Is this an accurate summary of the changes?
- Anywhere that called into old Settings.js, make it pull values from redux.
- Web does all its storing of settings values now through redux instead of directly accessing localStorage via Settings.
- audio only, startWithAudioMuted, startWithVideoMuted were in the profile feature but have been moved to settings.

Reply:
- Yes
- Yes
- Yes, it was the case when you reviewed it, but based on a discussion with Lyubo yesterday, base/profile is now killed off and we only have base/settings that handles all the in-app settings.